pages/1.13/cli/command-reference/dcos-node/dcos-node-dns/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: layout.pug
+navigationTitle:  dcos node dns
+title: dcos node dns
+menuWeight: 6
+excerpt: Viewing DC/OS node information
+enterprise: false
+---
+
+# Description
+
+The `dcos node dns` command allows you to view your DC/OS cluster's domain name service configuration.
```

Comment:
this commands looks like ti is displaying dns configuration specific for the node, why is it under node and not under network

Reply:
I think this is out of the scope of this PR, can you please ping @masonse to see if we should move this subcommand and start deprecating `dcos node dns` for 1.14?

Reply:
https://jira.mesosphere.com/browse/DCOS-51447 created